### PR TITLE
Jhonson Algorithm in Kotlin

### LINKED ISSUE #2206: Johnson Algorithm
- [x] C 
- [ ] C++ - @Aakansha99
- [x] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @BhardwajShweta25
- [ ] Java - @Neelaksh-Singh
- [ ] JS - @thakran14
- [x] Kotlin - @ritish099
- [x] Php - @ritish099
- [ ] Python - @BishalKrChanda
- [ ] Readme - @AdityaSingh17
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #2206

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Jhonson Algorithm in Kotli

#### Languages Used:
- 
- 

#### Files Added:
- Jhonson_Algorithm.kt

Thanks!

### CODE REVIEW

**Comment by ritish099** [comment]
Mam @asha15 , Check it out

File: Jhonson_Algoritm/Jhonson_Algorithm.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        c = readLine() !!.toInt()
+        adj[i][j] = c
+        cost[i][j] = c
+    }
+
+    for (i in 1..vert)
+        for (j in 1..vert)
+        {
+            if (adj[i][j] == 0 && i != j)
+            //If its not a edge put infinity
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.kt:32]: please correct the indentation

File: Jhonson_Algoritm/Jhonson_Algorithm.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                adj[i][j] = inf;
+        }
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.kt:34]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.kt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    for (k in 1..vert)
+        for (i in 1..vert)
+            for (j in 1..vert)
+            //Finding the minimum
+            //find minimum path from i to j through k
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.kt:39]: please correct the indentation

File: Jhonson_Algoritm/Jhonson_Algorithm.kt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                if ((adj[i][k] + adj[k][j]) > adj[i][j])
+                    adj[i][j] = adj[i][j]
+                else
+                    adj[i][j] = (adj[i][k] + adj[k][j])
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.kt:43]: keep a line space after this

File: Jhonson_Algoritm/Jhonson_Algorithm.kt [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    for (i in 1..vert)
+    {
+        for (j in 1..vert)
+        {
+            if (adj[i][j] != inf)
+                print("${adj[i][j]} ");
+        }
+        println(" ");
+    }
+}
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson_Algorithm.kt:55]: keep a line after this

**Comment by ritish099** [comment]
Mam @asha15 , Check it out !

**Pull request merged**